README.md

```diff
@@ -112,6 +112,20 @@ such as restricting discovered features with the --label-whitelist option._
 | SSE4.2         | Streaming SIMD Extensions 4.2 (SSE4.2)
 | SGX            | Software Guard Extensions (SGX)
 
+### Arm64 CPUID Features (Partial List)
+
+| Feature name   | Description                                                  |
+| :------------: | :----------------------------------------------------------: |
+| AES            | Announcing the Advanced Encryption Standard
```

Comment:
Is this AES different from x86 AES? If not please remove it from this list. Same comment holds for rest of the feature list.

Reply:
Hi @balajismaniam 
For same features, There are different names on the two platforms.
In x86 platform, it is defined as following:
AESNI:       "AESNI",       // Advanced Encryption Standard New Instructions
In Arm platform:
AES            | Announcing the Advanced Encryption Standard
For more reference, please see the file as reference:
https://github.com/lubinsz/cpuid/blob/master/cpuid.go

Reply:
Got it. Thanks.